.bowerrc

```diff
@@ -0,0 +1 @@
+{ "directory": "public/lib" }
```

Comment:
I'm using npm for dev automation related dependencies, which are not served to browsers, and bower for "live" dependencies, which are served, so they need to go under `public`.